utils/src/main/kotlin/Logger.kt

```diff
@@ -48,3 +53,73 @@ inline fun <reified T : Any> T.logOnce(level: Level, supplier: () -> String) {
         log.log(statement.second, statement.third)
     }
 }
+
+/**
+ * A log [Level] for logging performance information. The int value of the level is between [Level.INFO] and
+ * [Level.DEBUG].
+ */
+val PERFORMANCE = Level.forName("PERFORMANCE", 450)
```

Comment:
How about replacing the hard-coded `450` with `(Level.INFO.intLevel() + Level.DEBUG.intLevel()) / 2` to emphasize what is documented?

Reply:
It seems unnecessarily complex to me, given that it is extremely unlikely that the values in log4j will ever change.

Reply:
I agree it's more complex and probably not necessary as the int values won't change, but I just found it to be nice 😁 Anyway, it's completely optional.